src/data/ability.ts

```diff
@@ -6315,7 +6315,9 @@ export function initAbilities() {
       .attr(PostTurnStatusHealAbAttr, StatusEffect.TOXIC, StatusEffect.POISON)
       .attr(BlockStatusDamageAbAttr, StatusEffect.TOXIC, StatusEffect.POISON),
     new Ability(Abilities.ADAPTABILITY, 4)
-      .attr(StabBoostAbAttr),
+      .attr(StabBoostAbAttr, (user, target, move) => {
+        return ![ Moves.STRUGGLE ].includes(move.id);
+      }),
```

Comment:
Iirc this will work (didn't check in VSCode).
```suggestion
      .conditionalAttr((_user, _target, move) => ![ Moves.STRUGGLE ].includes(move.id), StabBoostAbAttr),
```